config/applications.json

```
@@ -2638,5 +2638,13 @@
 		"description": "The customization marketplace for Windows programs",
 		"link": "https://windhawk.net",
 		"winget": "RamenSoftware.Windhawk"
+	},
+	"WPFInstallForceAutoHDR": {
+		"category": "Games",
```

```suggestion
		"category": "Utilities",
```
The addition looks alright to me, but may I give you a suggestion on a particular field (the `category` field to be specific)?

The [tool's Repo Page](https://github.com/7gxycn08/ForceAutoHDR) shows that it's indeed meant to be used as a Utility/Tool rather than a Game, I think it's more fitting to change the Category for it from `Games` (what's currently at) to `Utilities`.

---

Yep my bad thought games related tools should go under "Games".
Updated commit.